templates/node-cli/lib/parser.js.twig

```diff
@@ -0,0 +1,191 @@
+                        break;
+                    default:
+                        rowValues.push(`array(${row[key].length})`);
+                        break;
+                }
+            } else if (typeof row[key] === 'object') {
+                rowValues.push("object");
+            } else {
+                rowValues.push(row[key]);
+            }
+        }
+        table.push(rowValues);
+    });
+    console.log(table.toString());
+}
+
+const drawJSON = (data) => {
+    console.log(JSON.stringify(data, null, 2));
+}
+
+const parseError = (err) => {
+    if(cliConfig.verbose) {
+        console.error(err);
+    }
+
+    error(err.message);
+
+    process.exit(1)
+}
+
+const actionRunner = (fn) => {
+    return (...args) => fn(...args).catch(parseError);
+}
+
+const parseInteger = (value) => {
+    const parsedValue = parseInt(value, 10);
+    if (isNaN(parsedValue)) {
+        throw new commander.InvalidArgumentError('Not a number.');
+    }
+    return parsedValue;
+}
+
+const parseBool = (value) => {
+    if (value === 'true') return true;
+    if (value === 'false') return false;
+    throw new commander.InvalidArgumentError('Not a boolean.');
+}
+
+const log = (message) => {
+    console.log(`${chalk.cyan.bold("ℹ Info")} ${chalk.cyan(message ?? "")}`);
+}
+
+const success = (message) => {
+    console.log(`${chalk.green.bold("✓ Success")} ${chalk.green(message ?? "")}`);
+}
+
+const error = (message) => {
+    console.error(`${chalk.red.bold("✗ Error")} ${chalk.red(message ?? "")}`);
+}
+
+const logo = "\n" +
+    "    _                            _ _           ___   __   _____ \n" +
+    "   /_\\  _ __  _ ____      ___ __(_) |_ ___    / __\\ / /   \\_   \\\n" +
+    "  //_\\\\| '_ \\| '_ \\ \\ /\\ / / '__| | __/ _ \\  / /   / /     / /\\/\n" +
+    " /  _  \\ |_) | |_) \\ V  V /| |  | | ||  __/ / /___/ /___/\\/ /_  \n" +
+    " \\_/ \\_/ .__/| .__/ \\_/\\_/ |_|  |_|\\__\\___| \\____/\\____/\\____/  \n" +
+    "       |_|   |_|                                                  \n" +
+    "\n";
+
+const commandDescriptions = {
+    "account": `The account command allows you to authenticate and manage a user account.`,
+    "avatars": `The avatars command aims to help you complete everyday tasks related to your app image, icons, and avatars.`,
+    "database": `The database command allows you to create structured collections of documents, query and filter lists of documents.`,
+    "deploy": `The deploy command provides a convenient wrapper for deploying your functions and collections.`,
+    "functions": `The functions command allows you view, create and manage your Cloud Functions.`,
+    "health": `The health command allows you to both validate and monitor your {{ spec.title|caseUcfirst }} server's health.`,
+    "init": `The init command helps you initialize your {{ spec.title|caseUcfirst }} project, functions and collections`,
+    "locale": `The locale command allows you to customize your app based on your users' location.`,
```

Comment:
Can we generate some of this block?

Reply:
Unfortunately the description of the services is not part of the swagger spec. Unless we add it to the spec itself, it doesn't seem like we can generate this .

Reply:
I was sure we did it. See the `tags` attribute here: https://raw.githubusercontent.com/appwrite/appwrite/master/app/config/specs/swagger2-0.12.x-server.json

Reply:
![Screenshot 2022-02-24 at 2 05 00 PM](https://user-images.githubusercontent.com/20852629/155503080-aad9112a-dec1-420a-bdb9-e49905443973.png)
I believe tags give information about an endpoint and not about the entire service.

Reply:
These descriptions are for the entire service as a whole